docs/can-canjs/can-api.md

```diff
@@ -277,17 +292,56 @@ console.log(areSomeComplete); //-> false
 ```
 @codepen
 
-<details>
-<summary>Infrastructure APIs</summary>
+</details>
+
+## Typed properties
+
+[can-type] is helpful to define typed properties for models and components by doing the following:
+- Validates properties `value` types
+- Converts a value to a type
+- Allows `undefined` / `null` to be values 
+- Converts a value to the correct type if not `null` / `undefined`
+
+Also, you can use the defined custom types as value types, for example the following defines a `User` type 
+that has a `person` property with `Person` type:
 
 ```js
-const fullName = new Observation(() => {
-    return person.first + " " + person.last;
-});
-```
+import { ObservableObject, type } from "can";
 
-</details>
+class Person extends ObservableObject {
```

Comment:
I think this example can be simplified to something similar to the first example in https://www.bitovi.com/blog/canjs-6.0#new-type-system.

I would just show creating an instance of an observable that has
- a strict property
- a property that is converted with `type.convert`
- a property that is `null` or `undefined`

Then show setting a property to the wrong type and the error that is thrown.